modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1000,11 +1002,21 @@ inline void fill_codec_context(AVCodecContext * enc, AVDictionary * dict)
     }
 }
 
+static bool isThreadSafe() {
+    const bool threadSafe = utils::getConfigurationParameterBool("OPENCV_FFMPEG_IS_THREAD_SAFE", false);
```

Comment:
`static` maybe?

Reply:
Will this make any difference if the the scope of `threadSafe` is limited to `isThreadSafe`?

Reply:
It depends on what we need.
- `static` - make variable with "global" lifetime and execute rhs expression once
- no `static` - read configuration on every call. We should not do that for performance sensitive cases, but there is no performance problem in the current case.